notebooks/logging-examples/logging-experiment-failures.ipynb

```diff
@@ -0,0 +1,157 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3b466d92",
+   "metadata": {},
+   "source": [
+    "## Distinguishing Failed Experiments with Rubicon \n",
+    "Modeling runs can fail for a number of reasons. When logging with rubicon_ml, a failed run may result in an empty or incomplete experiment. In this example, we'll walk through how to handle such experiments."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4f58990",
+   "metadata": {},
+   "source": [
+    "First lets simulate the problem. To do this we'll create an estimator that will fail on it's `fit()` 30% of the time.  We will consider any experiment that has a `none` value for `self.state`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "f9a506c3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.base import BaseEstimator\n",
+    "import random\n",
+    "\n",
+    "class BadEstimator(BaseEstimator):\n",
+    "    def __init__(self, state=None):\n",
+    "        super().__init__()\n",
+    "        self.state=None\n",
```

Comment:
we don't need to set this to `None` in the constructor. learned attributes like `self.state_` are either available as attributes after a call to `fit` or don't exist as attributes on the object at all

for example, pre-`fit` `bad_estimator_instance.state_` would error